db/migrate/20160703212040_add_org_admin_flag_to_memberships.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrgAdminFlagToMemberships < ActiveRecord::Migration
+  def change
+    add_column :memberships, :org_admin, :boolean
```

Comment:
A different mechanism was setup for creating roles. Rather than using a boolean field on the User model, a bitmask field was created (look in `/app/models/user.rb`). You can add a new role by adding the value to the ROLES constant 

Upon reflection, I think it would be good if we used CanCanCan gem + [Rolify](https://github.com/RolifyCommunity/rolify#4-add-a-role-to-a-user) for roles & authorization. I'm using Rolify in another project right now and it's pretty easy to use. Plus, it allows you to create roles assigned globally, to a resource instance (e.g. edit Organization with ID of 17), or to a resource class (e.g. edit all Organizations)